opcua/server/internal_subscription.py

```diff
@@ -59,15 +59,16 @@ def trigger_datachange(self, handle, nodeid, attr):
     def _modify_monitored_item(self, params):
         with self._lock:
             for mdata in self._monitored_items.values():
-                result = ua.MonitoredItemCreateResult()
+                result = ua.MonitoredItemModifyResult() # main thing that was wrong
```

Comment:
Before merging comment should be removed.

Also I am not sure that comments under are also really needed...